Models/SSBTaxAssessment/POCO/Dimension.cs

```diff
@@ -0,0 +1,108 @@
+using System.Collections.ObjectModel;
+using System.Text.Json.Serialization;
+
+namespace Backend.Models.SSBTaxAssessment.POCO
+{
+    public class Dimension
+    {
+        [JsonPropertyName("Region")] public Region Region { get; set; }
+
+        [JsonPropertyName("Alder")] public Alder Alder { get; set; }
+
+        [JsonPropertyName("Tid")] public Tid Tid { get; set; }
+
+        [JsonPropertyName("ContentsCode")] public ContentsCode ContentsCode { get; set; }
+    }
+
+    public class Region
```

Comment:
File is called `Dimension.cs` implying that that is the only class contained within

Reply:
This was an auto-generated file, but is deleted now.